types/index.d.ts

```diff
@@ -1,3 +1,5 @@
+export function setupCanvasMock() {}
```

Comment:
I'm not very familiar with all the details of TypeScript, but with the `window` argument added, does this line also need to change?

Reply:
updated!